srcpkgs/apparmor/template

```diff
@@ -15,8 +15,9 @@ short_desc="Mandatory access control to restrict programs"
 maintainer="Olivier Mauras <olivier@mauras.ch>"
 license="GPL-2.0-only, LGPL-2.1-only"
 homepage="https://gitlab.com/apparmor/apparmor"
+changelog="https://gitlab.com/apparmor/apparmor/-/wikis/Release_Notes_3.0.1"
```

Comment:
`${version}`

Also wow having versioned release notes sucks :/

Reply:
Hmm, in this case it's having to update this manually vs having the URL not have variables.....

Reply:
I missed that, thanks. There is a changelog file in libraries/libapparmor/ChangeLog, but... it hasn't been updated in 12 years and it's an empty file. :)